subprojects/gradle/cd/src/main/kotlin/com/avito/ci/BuildStepListExtension.kt

```diff
@@ -123,6 +127,14 @@ open class BuildStepListExtension(
         configureAndAdd("lint", action)
     }
 
+    fun markReportAsSourceForTMS(closure: Closure<MarkReportAsSourceForTMSStep>) {
```

Comment:
Is that for backward compatibility?
As I understand a function with `Action` will be enough

Reply:
We use groovy in tests configuration, it's not working with `Action` only for some reason

Reply:
Are you sure? It should work https://docs.gradle.org/current/userguide/kotlin_dsl.html#groovy_closures_from_kotlin